ontology/chiro.md

```diff
@@ -22,6 +22,9 @@ products:
     name: "CHEBI Integrated Role Ontology main release in OWL format"
   - id: chiro.obo
     name: "CHEBI Integrated Role Ontology additional release in OBO format"
+publications: 
+   - id: https://chemrxiv.org/engage/chemrxiv/article-details/60c74d1f9abda21f6df8d3ce
```

Comment:
I am SUPER upset with the changes to chemRxiv, I will probably never use this preprint server again - they removed the identifiers from the webpage for viewing. Total nonsense. Instead we should solely use the DOI

```suggestion
   - id: https://doi.org/10.26434/chemrxiv.12591221
```